signac/contrib/project.py

```diff
@@ -176,12 +181,10 @@ def __init__(self, config=None, _ignore_schema_version=False):
 
     def __str__(self):
         """Return the project's id."""
-        return str(self.id)
+        return repr(self)
 
     def __repr__(self):
-        return "{type}.get_project({root})".format(
-            type=self.__class__.__name__, root=repr(self.root_directory())
-        )
+        return f'{self.__class__.__name__}("{self.root_directory()}")'
```

Comment:
Woop, this is a regression. This must call `repr(self.root_directory())` to add its own quotes and get the escaping correct for arbitrary paths. Then double-quotes are permitted around the outside f-string, required by style fixers when double quotes are not contained in the string.
```suggestion
        return f"{self.__class__.__name__}({repr(self.root_directory())})"
```